SU-HSDO/suhumsci
# feat (STN-779): Collection Component Spacing

# READY FOR REVIEW

## Summary
The work in this PR adds spacing at the bottom of collection components.

### Screenshot of collection components before spacing was added
Note that I added a background color of violet to collection components to clearly identify the spacing issue. This background color was only used for testing.

#### An example of no space between the collection component and text area beneath it.
![before collection above text area](https://user-images.githubusercontent.com/12678977/119876824-4abc0380-bef6-11eb-8b60-78efcdfecb72.png)

#### An example of no space between multiple collection components and the headings beneath them.
![collection spacing before](https://user-images.githubusercontent.com/12678977/119876953-717a3a00-bef6-11eb-8f39-f1fdd463a1a4.png)

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. In you CLI, run `npm run test` and confirm our tests continue to pass.
2. The following examples of the collection component are available on staging: 
    * https://sparkbox-sandbox-stage.stanford.edu/collection-qa (includes style issues)
    * https://sparkbox-sandbox-stage.stanford.edu/collection-qa-2 (with realistic content)
3. I recommend syncing with staging so that you can use test the work in this PR against the examples on staging. Or you can create a new flexible page and add content and components similar to what has been created on staging.
4. - [x] Confirm that Collection components have a margin-bottom applied so there is space between a Collection component and the content or component below it.
